templates/projects/empty-server/data.js

```diff
@@ -81,7 +81,6 @@ template.server = {
       cors: false,
       handleErrors: false,
     }},
-    { name: 'legacyExplorer', value: false },
```

Comment:
This should be parameterized. For apps based on LB 3.0 and newer, the entry should not be there. For apps based on LB 2.x, the entry should be preserved.